src/test/java/uk/gov/justice/hmpps/prison/executablespecification/steps/AuthTokenHelper.java

```diff
@@ -77,6 +78,7 @@ public AuthTokenHelper(final JwtAuthenticationHelper jwtAuthenticationHelper) {
         tokens.put(String.valueOf(AuthToken.SMOKE_TEST), createSmokeTestUser());
         tokens.put(String.valueOf(AuthToken.REF_DATA_MAINTAINER), createRefDataMaintainerUser(true));
         tokens.put(String.valueOf(AuthToken.REF_DATA_MAINTAINER_NO_WRITE), createRefDataMaintainerUser(false));
+        tokens.put(String.valueOf(AuthToken.UNAUTHORISED_USER), createUnauthorisedUser());
     }
```

Comment:
Maybe there is a better name for this user without any roles or scope - I needed it to get a @verifyBookingAccess to fail